app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchmessage } from '../redux/message/api';
+
+const Greeting = () => {
+  const message = useSelector((state) => state.messageReducer.message);
+  // console.log('message => ', message);
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchmessage());
+  }, []);
```

Comment:
- [ ] You should fix this error.

![image](https://user-images.githubusercontent.com/38635158/159363394-13efccbd-8daf-4570-a7b1-4f83606f6124.png)